app/src/main/java/com/ataulm/whatsnext/FilmSummary.kt

```diff
@@ -1,22 +1,13 @@
 package com.ataulm.whatsnext
 
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
 data class FilmSummary(
         val ids: Ids,
         val name: String,
         val year: String?,
-        val runtimeMinutes: Int?,
-        val tagline: String?,
-        val description: String?,
         val poster: Images,
-        val backdrop: Images,
-        val genres: List<String>,
-        val cast: List<Actor>,
-        val crew: List<Contributor>
```

Comment:
none of this stuff gets returned in a film summary (representation of a film in most APIs)